pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "volttron-sqlite-historian"
-version = "0.2.0rc"
+version = "1.0.0"
```

Comment:
Weren't we using 1.0 to be "pre-auth update"

Reply:
I just updated major version for all the existing library and agent to indicate they are backward incompatible